site/docs/install-ubuntu.md

```diff
@@ -68,28 +68,30 @@ export PATH="$PATH:$HOME/bin"
 
 You can also add this command to your `~/.bashrc` file.
 
-_<sup>1</sup>Note that Bazel includes an embedded JDK 8, which can be used even if a JDK is already
+_<sup>1</sup>Note that Bazel includes an embedded JDK, which can be used even if a JDK is already
 installed. `bazel-<version>-without-jdk-installer-linux-x86_64.sh` is a version of the installer 
-without embedded JDK 8. Only use this installer if you already have JDK 8 installed._
+without an embedded JDK. Only use this installer if you already have JDK 8 or later installed._
```

Comment:
No, later JDKs are not supported, only JDK 8 is.

Reply:
Updated and added an explicit note to reflect this.